x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_events_query.test.ts

```diff
@@ -100,13 +88,22 @@ describe('create_signals', () => {
               {},
               {
                 bool: {
-                  filter: [
+                  should: [
```

Comment:
The diff is hard to read, but the change here is really just removing the extra levels of nesting.